src/_tutorials/dart-vm/httpserver.md

```diff
@@ -98,11 +98,11 @@ with the string `Hello, world!`
 At the command line, run the `hello_world_server.dart` script.
 You will see the following:
```

Comment:
Let's delete the second sentence, changing this to:

...script:

Reply:
done